electron-app/src/server/websites/mastodon/mastodon.service.ts

```diff
@@ -60,7 +60,7 @@ export class Mastodon extends Megalodon {
   getInstanceSettings(accountId: string) {
     const instanceInfo: MastodonInstanceInfo = this.getAccountInfo(accountId, INFO_KEY);
 
-    this.maxCharLength = instanceInfo?.configuration?.statuses?.max_characters ?? 500;
+    this.MAX_CHARS = instanceInfo?.configuration?.statuses?.max_characters ?? 500;
```

Comment:
issue: Multiple instances will clobber the global value since they all share it.
In v4 this wouldn't be a problem, but in v3 we need to store vals like this in the individual account info store.

Reply:
Resolved